src/aavrutiDJun20/PalindromeString.java

```diff
@@ -0,0 +1,25 @@
+//Print whether given String is Palindrome or not
+package aavrutiDJun20;
+
+public class PalindromeString {
+	
+	void palindromeStringOrNot(String input) {
+		String palindromeInput = "";
+		
+		for(int index=input.length()-1;index>=0;index--) {
+			palindromeInput += String.valueOf(input.charAt(index));
+		}
+		
+		if(palindromeInput.equalsIgnoreCase(input))
```

Comment:
I believe we should use equals(), if interviewer ask to make it case-insensitive we can use equalIgnoreCase